packages/component/src/chip/Chip.tsx

```diff
@@ -0,0 +1,98 @@
+import {
+	ChipProps,
+	ColorTokens,
+	CornerRadiusTokens,
+	FontFamilyTokens,
+	FontSizeTokens,
+	IconSizeTokens,
+	IconTokens,
+	SpaceTypeTokens,
+	StackAlignType,
+	StackType,
+} from '@blue-learn/schema';
+import React, { memo } from 'react';
+import { View, Text } from 'react-native';
+import ThemeProvider from '@blue-learn/theme';
+import Typography from '../typography/Typography';
+import Icon from '../icon/Icon';
+import Space from '../space/Space';
+import Stack from '../stack/Stack';
+
+/**
+ * @description
+ * Stack supports children as React Component and widgetItems as WidgetItem Type[order by children -> widgetItems]
+ * Note: renderItem should be passed to render WidgetItem
+ **/
+const Chip: React.FC<ChipProps> = ({
+	label = 'Chip',
+	bgColor,
+	labelColor = ColorTokens.Grey_100,
+	fontSize = FontSizeTokens['2xs'],
+	borderRadius = CornerRadiusTokens.BR4,
+	iconAlignment = 'left',
```

Comment:
can we add this props to icon default props? since have seen this type in button also

Reply:
alignment is with respect to text. so how we can add that prop in icon default props